lib/error_tracker/web/live/show.ex

```diff
@@ -121,4 +121,7 @@ defmodule ErrorTracker.Web.Live.Show do
     |> limit(^num_results)
     |> Repo.all()
   end
+
+  defp error_has_source_info?(%Error{source_function: "-", source_line: "-"}), do: false
+  defp error_has_source_info?(%Error{}), do: true
```

Comment:
Maybe we can move this function to `ErrorTracker.Error.has_source_info?` so we can reuse it both in show and dashboard templates. We are currently using it in the show template but doing the check manually in the dashboard template.

What do you think?

Reply:
Updated in https://github.com/elixir-error-tracker/error-tracker/pull/70/commits/e658416b4a8597cad0377c99c76cfc17d6743c9b